assets/src/blocks/Covers/CampaignCovers.js

```diff
@@ -1,59 +1,51 @@
 import { IMAGE_SIZES } from './imageSizes';
-const { __ } = wp.i18n;
 
-export const CampaignCovers = ({ covers, initialRowsLimit, row, loadMoreCovers, inEditor = false }) => {
-  const amountPerRow = 3;
-  const showLoadMore = !!initialRowsLimit && covers.length > amountPerRow * row;
+const { __ } = wp.i18n;
 
-  return (
-    <>
-      <div className='thumbnail-largeview-container'>
-        {covers.map((cover, index) => {
-          const { href, image, alt_text, name, src_set } = cover;
-          const hideCover = !!initialRowsLimit && index >= row * amountPerRow;
+export const CampaignCovers = ({
+  covers,
+  initialRowsLimit,
+  row,
+  inEditor = false,
+  isCarouselLayout,
+  amountOfCoversPerRow,
+}) => (
+  <div className='covers'>
+    {covers.map((cover, index) => {
+      const { href, image, alt_text, name, src_set } = cover;
+      const hideCover = !isCarouselLayout && !!initialRowsLimit && index >= row * amountOfCoversPerRow;
 
-          if (hideCover) {
-            return null;
-          }
+      if (hideCover) {
+        return null;
+      }
 
-          const campaignLink = inEditor ? null : href;
+      const campaignLink = inEditor ? null : href;
```

Comment:
I think it's ok to keep it as is for now, it's more consistent with the other Covers styles and it might be useful in the future since we will eventually update the [designs for the Campaign Covers](https://p4-designsystem.greenpeace.org/05f6e9516/p/66c4dc-covers/b/937749)